hicommonwealth/commonwealth
# Updates chat application

## Description

Makes the chat app available again, but hidden from the sidebar. Updates WebSockets interfaces that have changed since the WebSocket server was generalized.

## Motivation and Context
Preparing for future work.

## How has this been tested?
- Send messages, ensure they appear across multiple clients
- Ensure scrollback appears when chat is loaded
- Needs more testing around chat server interfacing the with the database schema, to make sure that all checks for message visibility still function as expected
- We may want to merge the chat server into the commonwealth repo to make maintainability easier, since it implicitly relies on our models

## Have proper tags been added (for bug, enhancement, breaking change)?

## Does this PR affect any server routes?
- [ ] yes, and they are tested: [enter the % coverage here]
- [ ] yes, and they are not tested: [enter the % coverage here]
- [ ] yes, but I did not run tests
